### PR TITLE
Update auth configuration for service portal

### PR DESCRIPTION
**Pull request by valurefugl**
# Suggested changes to auth configuration

## What

- Enable silent renew
- Store user in session storage
- Only re-authenticate if no user in session storage
- offline_access scope not requested (refresh tokens not needed)
- Add post_logout_redirect_uri

## Why

Streamline login and logout flows. Harden security.

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ x] Formatting passes locally with my changes
- [ x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/service-portal/src/utils/userManager.ts [hunk 1]
```diff
@@ -11,17 +11,18 @@ const settings: UserManagerSettings = {
...
+  // eslint-disable-next-line @typescript-eslint/camelcase
   response_type: 'code',
   revokeAccessTokenOnSignout: true,
   loadUserInfo: true,
-  automaticSilentRenew: false,
-  scope: 'openid profile offline_access api_resource.scope',
-  userStore: new WebStorageStateStore({ store: new InMemoryWebStorage() }),
+  automaticSilentRenew: true,
+  scope: 'openid profile api_resource.scope',
+  userStore: new WebStorageStateStore({ store: window.sessionStorage }),
```
Comment (obmagnusson) [inline apps/service-portal/src/utils/userManager.ts:25]: Why the change to session storage? Wouldnt this be vulnarable to XSS?

Comment (valurefugl) [reply apps/service-portal/src/utils/userManager.ts:25]: Potentially yes, and we should mitigate that risk. The upside is that we don't need to contact auth server on refresh.
Note that the in-memory storage also makes the tokens accessible in the browser.

File: apps/service-portal/src/hooks/useAuth/useAuth.ts [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import { useStore } from '../../store/stateProvider'
 import { ActionType } from '../../store/actions'
 import { userManager } from '../../utils/userManager'
+import { User } from '@sentry/react'
```
Comment (obmagnusson) [inline apps/service-portal/src/hooks/useAuth/useAuth.ts:4]: This is not used.

File: apps/service-portal/src/utils/userManager.ts (hunk 1 above)
Comment (obmagnusson) [reply apps/service-portal/src/utils/userManager.ts:25]: Yes but the risk I would guess is less when dealing with In-memory storage.  But maybe the upside outweighs the potentential risk. Am assuming this will or has been cleared with Syndis.

**Review by obmagnusson — CHANGES_REQUESTED** [review]
Miniscule changes

File: apps/service-portal/src/utils/userManager.ts (hunk 1 above)
Comment (valurefugl) [reply apps/service-portal/src/utils/userManager.ts:25]: Haven't cleared with Syndis but we should. As I see it the tokens are either stored in the browser or not, in-memory storage might be a bit more obscure but the tokens are still there.

File: apps/service-portal/src/hooks/useAuth/useAuth.ts (hunk 2 above)
Comment (valurefugl) [reply apps/service-portal/src/hooks/useAuth/useAuth.ts]: Fixed.

**Review by obmagnusson — APPROVED** [review]
LGTM
